src/loaders/api/apiAsync.js

```diff
@@ -66,7 +64,8 @@ export function setAPI (agentIdentifier) {
     const agentInfo = getInfo(agentIdentifier)
     if (!agentInfo.beacon) return
 
-    var url = scheme + '://' + agentInfo.beacon + '/1/' + agentInfo.licenseKey
+    const scheme = (getConfigurationValue(agentIdentifier, 'ssl') === false) ? 'http' : 'https'
+    let url = `${scheme}://${agentInfo.beacon}/1/${agentInfo.licenseKey}`
```

Comment:
Does this need to support the proxy configuration?

Reply:
0.017% > 0% Personally, I don't understand the use-case for this API but it is being used. If those customer's using this API try to use proxy, it's not going to work like it should.

Reply:
Updated to mirror harvest